be/src/exec/vectorized/hdfs_scan_node.cpp

```diff
@@ -521,6 +532,7 @@ Status HdfsScanNode::close(RuntimeState* state) {
 Status HdfsScanNode::set_scan_ranges(const std::vector<TScanRangeParams>& scan_ranges) {
     for (const auto& scan_range : scan_ranges) {
         _scan_ranges.emplace_back(scan_range.scan_range.hdfs_scan_range);
+        COUNTER_UPDATE(_scan_files_counter, 1);
```

Comment:
_scan_ranges_counter？
There may be multiple scan ranges corresponding to one file.

Reply:
maybe we should have two counter. One is `_scan_files_counter` which means how many different files. Another is `_scan_ranges_counter` which means how many scan ranges.

Reply:
Yes, `scan_ranges_counter` is better.